src/UI/templates/default/Glyph/tpl.glyph.html

```diff
@@ -0,0 +1,80 @@
+<a class="glyph<!-- BEGIN highlighted --> highlighted<!-- END highlighted -->" <!-- BEGIN with_action -->href="{ACTION}"<!-- END with_action --> aria-label="{LABEL}">
+
+
+<!-- BEGIN settings -->
+<span class="glyphicon glyphicon-cog" aria-hidden="true"></span>
+<!-- END settings -->
+
+<!-- BEGIN collapse -->
+<span class="glyphicon glyphicon-triangle-bottom" aria-hidden="true">Issue: New Bootstrap Version (3.3.6) Needed</span>
+<!-- END collapse -->
+
+<!-- BEGIN expand -->
+<span class="glyphicon glyphicon-triangle-right" aria-hidden="true">Issue: New Bootstrap Version (3.3.6) Needed</span>
+<!-- END expand -->
+
+<!-- BEGIN add -->
+<span class="glyphicon glyphicon-plus-sign" aria-hidden="true"></span>
+<!-- END add -->
+
+<!-- BEGIN remove -->
+<span class="glyphicon glyphicon-minus-sign" aria-hidden="true"></span>
+<!-- END remove -->
+
+<!-- BEGIN up -->
+<span class="glyphicon glyphicon-circle-arrow-up" aria-hidden="true"></span>
+<!-- END up -->
+
+<!-- BEGIN down -->
+<span class="glyphicon glyphicon-circle-arrow-down" aria-hidden="true"></span>
+<!-- END down -->
+
+<!-- BEGIN back -->
+<span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
+<!-- END back -->
+
+<!-- BEGIN next -->
+<span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>
+<!-- END next -->
+
+<!-- BEGIN sortAscending -->
+<span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span>
+<!-- END sortAscending -->
```

Comment:
But the description already says "The Sort Ascending Glyph uses glyphicon-arrow-up.". This text will be displayed in the kitchen sink or not?

Sorry that I do not understand your second point in this special case. Why would you need to change a class, if you already have a separate class for each type of the glyph? You already can adress each glyphs style separately. Things like ::before and ::after will give you even more flexibility to change the design.

"I think designers need a fair chance to perform customization up to and including level 3.". As I wrote this is a trade-off between centralization/maintainability and flexibility. I do not see the need to be able to change the HTML of each glyph here. There will be always limitations. We won't be able to allow designers to change the HTML of every single table in ILIAS.